card/VERSION

```diff
@@ -1 +1 @@
-1.14.8s
+1.15.pre
```

Comment:
Mostly, I want a good way to refer to it when we do.  And totally agree about where we can refer to what.